Makefile

```diff
@@ -196,3 +196,11 @@ clean:
 	rm -rf ./tests/e2e/default.proxy
 	rm -rf ./bin/shellcheck*
 	find ./ -name "127.0.0.1:*" -o -name "localhost:*" -o -name "*.log" -o -name "agent-*" -o -name "*.coverprofile" -o -name "testname-proxy-*" -delete
+
+.PHONY: verify-go-versions
+verify-go-versions:
```

Comment:
Please get `verify-go-versions` included in the target `verify`, and you don't add a separate workflow check because there is an existing workflow check `static-analysis.yaml`
https://github.com/etcd-io/etcd/blob/3d4d4a0053915f701e7d1b75834b309364d65109/Makefile#L55

Reply:
Done